src/Paging.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from "react";
+import "./Paging.css";
+
+export default class Paging extends Component {
+  constructor() {
+    super();
+    this.state = { page: 1 };
+  }
```

Comment:
You don't have to use this constructor, you can achieve the same result by declaring state like this:
```
state = {
    page: 1
}
```